logging/lifecycle/lifecycle.go

```diff
@@ -16,8 +15,11 @@ import (
 	"time"
```

Comment:
"time" should be up with "os", followed by \n, then any eris dependencies, then another \n then external dependencies